### DISCUSSION TITLE
✅ Ability to Turn Off Notes

### DISCUSSION THREAD

**Discussion by arequestions** [elementor/elementor#25822] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Hi,
Can you please bring back the ability to enable/disable notes? Notes create a compatibility issue with another plugin that is essential to our site, and due to this conflict, we cannot delete users. Before, we were able to disable the Notes, so we were able to resolve the issue, but since the recent update on 04/16/2024, we can't disable the notes, and we can't delete users.  

### Describe the solution you'd like.

Bring back to enable/disable notes.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by brandonaaronson** [comment]
I'm going to comment so this hopefully brings it back above the spam.  I too, do not like the Notes feature.  I left a comment on Wordpress but I figure I'd put my 2¢ here as well. Being forced to use Notes is a big detriment to our team workflow.  We have other systems in place that we utilize that don't require us to have another thing to keep track of inside sites we manage. Please return the ability to turn off notes.

**Reply by arequestions** [reply]
Thank you for supporting this. We don't have a proper workaround, which is a big headache for our site.

**Comment by hyperjit** [comment]
+1. Give us an option to disable it.

**Comment by paradoxalyty** [comment]
Many of us use the keyboard shortcut Cmd+Shift+c to open developer mode in Chrome. Instead, the Elementor notes are sporadically activated.
Please change the shortcut.

**Comment by hamdan-Deb** [comment]
Please provide us the control to turn off notes.

**Comment by pmh90117** [comment]
+1

**Comment by iamfahrig** [comment]
+1

**Comment by kaelsense** [comment]
Plugins elbowing their way into the admin-bar without the possibility to disable this intrusion is just disrespectful to the users, and bad UI design. It smells like a bad Themeforest bloatware. You have to add back the possibility to disable Notes.

**Reply by arequestions** [reply]
Couldn't agree more...

**Comment by czannolli** [comment]
+1

**Comment by cangulostudios** [comment]
+1

**Comment by covency** [comment]
Up up, add the option to disable as soon as possible!

**Comment by churongcon** [comment]
+1

**Comment by mcrowell** [comment]
Please give us the ability to disable this feature. Not everyone needs/wants this feature.

**Comment by markoste** [comment]
Please give us the option to disable this feature.
It's never used by any projects I work.... therefore it is bloatware for me.

**Comment by myssonv** [comment]
This is really vital, I just don't understand why Elemeentor would take away the option to hide the Notes feature. At the moment, am hiding the notes icon and label on the [admin bar via CSS. ](https://aimode.co/disable-elementor-notes/)
![image](https://github.com/user-attachments/assets/6d09764e-4365-4f09-9f8c-af35e285d176)

**Comment by robaxx** [comment]
Same as the AI garbage... We need a simple switch to get rid of it.

**Reply by aksellno** [reply]
+1

**Comment by gerasimovdaniel** [comment]
In theory this should help to turn off notes:

```
add_action( 'elementor_pro/init', function() {
    if ( ! class_exists( '\ElementorPro\Modules\Notes\User\Capabilities' ) ) {
        return;
    }

    $admin = get_role( 'administrator' );

    // If the admin role doesn't exist, or the admin role doesn't have the read notes capability, return.
    if ( ! $admin || ! $admin->has_cap( \ElementorPro\Modules\Notes\User\Capabilities::READ_NOTES ) ) {
        return;
    }

    // Remove the notes capabilities from all roles.
    $roles = array_values( wp_roles()->role_objects );

    foreach ( $roles as $role ) {
        if ( ! ( $role instanceof \WP_Role ) ) {
            continue;
        }

        foreach ( \ElementorPro\Modules\Notes\User\Capabilities::all() as $cap ) {
            $role->remove_cap( $cap );
        }
    }
}, 9999 );
```

**Comment by E-Markers** [comment]
+1

**Comment by cgomezhumm** [comment]
+1 why are you forcing this on us Elementor team?

**Comment by fxaviers** [comment]
+1

**Comment by eknown** [comment]
+1

**Comment by jomarieminney** [comment]
+1000

**Comment by peterpleva** [comment]
Without CSS:
if (is_admin_bar_showing()) 
  add_action('admin_bar_menu', function($wp_admin_bar) {
      $wp_admin_bar->remove_node('elementor_notes');
  }, 201);

Also removing Elementor Debugger:
if (is_admin_bar_showing()) {
    add_action('elementor/init', function() {
        remove_action('admin_bar_menu', [ \Elementor\Plugin::instance()->inspector, 'add_menu_in_admin_bar' ], 201);
    });
    add_action('admin_bar_menu', function($wp_admin_bar) {
        $wp_admin_bar->remove_node('elementor_notes');
    }, 201);
}

**Comment by EnTonyH** [comment]
+1 to disable notes.

Very annoying because sometimes it cause overlapping shortcuts: Cmd+shft+c opens notes / opens developer mode in chrome.

**Reply by deepwaterpixel** [reply]
This is my biggest complaint about the feature -- I use the console 50x more than notes and it seems to be arbitrary when Notes decides to pop up instead of console.

**Reply by q6it** [reply]
have the same issue. Please give us an option to turn notes feature off or at least change the shortcut

**Comment by yratof** [comment]
Remove the bindings. Thats the easier solution, there shouldn't be a shortcut for notes that conflicts with the most obvious shortcut for developers.

**Reply by yratof** [reply]
Even worse, it's only shift C that is used:

```
  /**
   * @return {Object} All the shortcuts of the component.
   */
  defaultShortcuts() {
    return {
      toggle: {
        keys: 'shift+c',
        exclude: ['input', 'textarea']
      }
    };
  }
```

**Reply by q6it** [reply]
which file is that? So i can remove this annoying binding.

**Comment by vgauther** [comment]
+1

**Comment by darthquack** [comment]
+999. Really annoying

**Comment by bikrrr** [comment]
+1. Totally agree—this is really frustrating. I’ve moved away from Elementor for new sites, but I still have one that depends on it, and migrating would be a nightmare. 😞

**Comment by vgauther** [comment]
+1 - Should be done already

**Comment by raabmar** [comment]
+1

**Comment by vayurobins** [comment]
Agree, ability to completely disable Notes is important. It's frustrating that is there.

**Comment by blakemiller99** [comment]
+1 here. 

At least give us the ability to remove from the admin bar if we chose to enable / use notes. Many other plugins do this, such as Gravity Forms and Independent Analytics.

**Comment by q6it** [comment]
+1

**Comment by pkosciak** [comment]
+1

**Comment by thomaspe76** [comment]
Let us disable that!

**Comment by clarkewd** [comment]
+1 

possible solution- add to `functions.php` for elementor pro...  (ymmv)

    add_action('wp_print_scripts', function (){
        wp_dequeue_script('elementor-pro-notes');
        wp_dequeue_script('elementor-pro-notes-app-initiator');
    });

for non-pro possibly dequeue `elementor-v2-editor-notes`  although seems to cause errors in the dev console...

also, looks like they changed the notes shortcut to just `shift+c` although mine was still activating with `ctrl+shift+c`:


![Screen Region 2025-07-01 at 16 37 36](https://github.com/user-attachments/assets/27e50ec0-d8f4-4fb6-9fc2-7d60743015e3)


![Screen Region 2025-07-01 at 16 37 46](https://github.com/user-attachments/assets/ddd94b19-d22a-41fd-9103-b656346d61ec)

**Reply by vayurobins** [reply]
This works, so thanks for this solution. The "Notes" menu item in the top admin bar is still visible though :-)

**Comment by jmzolezzi** [comment]
Having a decluttered UI and try to make Elementor is fast as possible, please allow us to disable Notes feature.

**Comment by 8titi8** [comment]
+1
Why remove this option that was used by many to improve performance within the builder ?!

**Comment by nicholaszein** [comment]
Hello everyone!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.21.0! 🥳 

You can deactivate Elementor Notes by going to your WordPress **Dashboard** → **Users** → **Profile** → **Elementor Notes** → <kbd>Enable Elementor Notes notifications</kbd>.

For more information please check the [documentation](https://elementor.com/help/notes-feature/#:~:text=Delete-,How%20to%20grant%20Notes%20access%20to%20site%20users,-To%20ensure%20each).

Cheers 🥂
